STEP/Program.cs

```diff
@@ -29,17 +29,23 @@ private static void Main(string[] args)
 
         try
         {
+            Console.WriteLine("Performing syntactic analysis...");
             STEPParser.ProgramContext tree = parser.program(); // Parse the input starting at the "program" rule.
 
+            // TODO: error listener
+
```

Comment:
I like this: Det er ret fint det her med at printe hvad der sker, lidt som moderne loading :D

Reply:
hov skulle have været linjerne ovenfor

Reply:
hehe